pkg/controllers/managementuserlegacy/cis/clusterScanHandler.go

```diff
@@ -457,7 +457,7 @@ func (csh *cisScanHandler) ensureCleanup(cs *v3.ClusterScan) error {
 	// Delete the pod
 	podName := runnerPodPrefix + cs.Name
 	if e := csh.podClient.Delete(podName, &metav1.DeleteOptions{}); e != nil && !kerrors.IsNotFound(e) {
-		err = multierror.Append(err, fmt.Errorf("cis: ensureCleanup: error deleting pod %v: %v", podName, e))
+		_ = multierror.Append(err, fmt.Errorf("cis: ensureCleanup: error deleting pod %v: %v", podName, e))
```

Comment:
This needs to be here, the err var above is collecting all errors. I don't think the linters likes the `if` check style used here since the other lines didn't complain and are doing essentially the same thing.